examples/gcp-rhel-vm/README.md

```diff
@@ -8,7 +8,7 @@
 
 ## Intel Red Hat Enterprise Linux GCP VM Example
 
-This module creates a Red Hat Enterprise Linux (RHEL) VM on the Intel Sapphire Rapids CPU. The virtual machine is created on an Intel Sapphire Rapids c3-standard-4 by default.
+This module creates a Red Hat Enterprise Linux (RHEL) VM on the Intel 5th Gen Emerald Rapids CPU. The virtual machine is created on an Intel Sapphire Rapids c3-standard-4 by default.
```

Comment:
Correct this "Intel Sapphire Rapids c3-standard-4"

Reply:
It's should say c4-standard-4 by default- Emerald Rapids.